src/html-templates/_includes/FirstColContent.tsx

```diff
@@ -35,7 +35,50 @@ export default function FirstColContent(): React.ReactElement {
           </AccordionItem>
         </AccordionWidget>
         <blockquote>Nulla quis lorem ut libero malesuada feugiat. Donec rutrum congue leo eget malesuada. Praesent sapien massa, convallis a pellentesque nec, egestas non nisi. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Donec velit neque, auctor sit amet aliquam vel, ullamcorper sit amet ligula. Sed porttitor lectus nibh. Curabitur non nulla sit amet nisl tempus convallis quis ac lectus. Curabitur aliquet quam id dui posuere blandit. Curabitur non nulla sit amet nisl tempus convallis quis ac lectus. Nulla porttitor accumsan tincidunt. Curabitur aliquet quam id dui posuere blandit.</blockquote>
+      </ContentBlock>
+      <ContentBlock>
         <h2>Header</h2>
+        <table>
+          <thead>
```

Comment:
Since `<thead>` and `<tbody>` are not required for accessibility we should consider removing them to simplify the table's HTML.

Reply:
I couldn't replicate the issue with the cursor. I've added a link and it uses the "pointer" cursor correctly. I've added it in the testing environment and JSX to test.